### PR TITLE
new option '--latest-signed'

### PR DESCRIPTION
**Pull request by M0ses**
This is a first draft to use only signed commits and needs some further enhancements to work inside containers

### CODE REVIEW

File: TarSCM/scm/git.py [hunk 1]
```diff
@@ -394,3 +420,68 @@ def check_url(self):
...
+            ['git', 'log', '--pretty=format:%H %G? %h %D', "--topo-order"],
+            cwd=self.clone_dir)
+
+        lines = result[1].split("\n")
+        while lines:
+            line = lines.pop(0)
+            commit = line.split(" ", 3)
+            if len(commit) > 3:
+                for tag in search_tags(commit[3]):
+                    tag = re.sub(",$", '',tag)
```
[the remark this answers is not in the document]
Comment (M0ses) [reply TarSCM/scm/git.py:474]: Not relevant - comma is part of an regex and dollar is end of line

**Comment by marcus-h** [comment]
On 2020-09-13 03:48:48 -0700, Frank Schreiner wrote:

<SNIP>

> +                    tag = re.sub(",$", '',tag)
>
> Not relevant - comma is part of an regex and dollar is end of line
>
I guess it is not about the regex but the "'',tag" part:)

**Comment by M0ses** [comment]
> I guess it is not about the regex but the "'',tag" part:)

THX for your hint - fixed

**Review by adrianschroeter — APPROVED** [review]
looks fine for me, it should not break anything existing so let's merge it and test it in a real setup in OBS....

**Pull request merged**